CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,91 @@
+  - Promote ClusterGroup and Group to v1beta1. ([#5181](https://github.com/antrea-io/antrea/pull/5181), [@GraysonWu])
+  - Promote ExternalIPPool API to v1beta1. ([#5176](https://github.com/antrea-io/antrea/pull/5176), [@hongliangl])
+  - Promote Tier API to v1beta1. ([#5172](https://github.com/antrea-io/antrea/pull/5172), [@GraysonWu])
+  - Promote Egress API to v1beta1. ([#5180](https://github.com/antrea-io/antrea/pull/5180), [@wenqiq])
+  - Promote Upgrade AntreaClusterNetworkPolicy and AntreaNativeNetworkPolicy API to v1beta1. ([#5186](https://github.com/antrea-io/antrea/pull/5186), [@GraysonWu])
+  - Promote Traceflow API to v1beta1. ([#5108](https://github.com/antrea-io/antrea/pull/5108), [@luolanzone])
+- Enable `proxyAll` by default for AntreaProxy on Windows because the kube-proxy userspace datapath has been removed since Kubernetes 1.26. ([#4980](https://github.com/antrea-io/antrea/pull/4980), [@XinShuYang])
+- Change default port range to `40000-41000` for NodePortLocal on Windows to avoid race condition when
+inserting Netnat rules. ([#5107](https://github.com/antrea-io/antrea/pull/5107), [@XinShuYang])
+- Remove the ClusterClaim CRD and upgrade the ClusterSet CRD version to v1alpha2, and enhance the ClusterSet controller to support ClusterSet version upgrade. ([#5001](https://github.com/antrea-io/antrea/pull/5001) [#5250](https://github.com/antrea-io/antrea/pull/5250), [@luolanzone])
+- Increase the controller QPS setting in Multi-cluster Controller to improve multi-cluster resource export performance, and increase the LabelIdentity controller worker count to improve its performance. ([#5099](https://github.com/antrea-io/antrea/pull/5099), [@GraysonWu])
+- Improve direct connections to Antrea apiserver in antctl with accessibility to Node ExternalIP and add a new `--insecure` option to support both secure and insecure connections. ([#5135](https://github.com/antrea-io/antrea/pull/5135), [@antoninbas])
+- Add two new fields to audit logs, including the "direction" of the NP rule (Ingress or Egress) and the reference of the Pod (<Namespace>/<Name>) to which the NP rule is applied. ([#5101](https://github.com/antrea-io/antrea/pull/5101), [@antoninbas])
+- Add a FlowExporter configuration toggle to antrea-agent for users to explicitly enable/disable flow exports. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
+- Add OpenAPI schema for the AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+- Add a validation schema for the matchLabels field of the ExternalIPPool CRD. ([#5284](https://github.com/antrea-io/antrea/pull/5284), [@tnqn])
+- Update short-names for AntreaNetworkPolicy as ANNP. ([#5081](https://github.com/antrea-io/antrea/pull/5081), [@qiyueyao])
+- Disable CGO for the antctl binary to avoid GLIBC dependency for cross-platform compatibility. ([#5240](https://github.com/antrea-io/antrea/pull/5240), [@XinShuYang])
```

Comment:
antctl in release asset has been built with CGO disabled long time ago. The change is for the binary built in antrea image, which is used for testing mainly. I feel no need to mention it to avoid confusion.